cnf-tests/testsuites/pkg/utils/consts.go

```diff
@@ -128,7 +128,7 @@ const (
 	// GatekeeperControllerDeploymentName contains the name of the gatekeeper-controller-manager deployment
 	GatekeeperControllerDeploymentName = "gatekeeper-controller-manager"
 	// GatekeeperOperatorDeploymentName contains the name of the gatekeeper-operator-controller-manager deployment
-	GatekeeperOperatorDeploymentName = "gatekeeper-operator-controller-manager"
+	GatekeeperOperatorDeploymentName = "gatekeeper-operator-controller"
```

Comment:
ok, but why? some context in the commit message would help understanding the rationale

Reply:
comment updated